src/language.js

```diff
@@ -238,13 +238,14 @@ const pages = {
     newBenthicAttributeLink: 'Propose New Benthic Attribute...',
   },
   projectInfo: {
-    title: 'Project Info',
-    newOrganizationNameLink: 'Suggest a new organization to MERMAID...',
     createOrganizationTitle: 'Suggest a new organization',
-    suggestionOrganizationHelperText: `If your organization is approved, it'll be automatically added to your project.`,
-    organizationsHelperText: `Type to search for an organization.`,
+    newOrganizationNameLink: 'Suggest a new organization to MERMAID...',
+    noNotes: 'no notes for this project',
```

Comment:
added noNotes becaue we are trying to put all the hardcoded text in the language file to make like easier for Al redoing the copy of the site. Teh rest of the diff is just alphabetization